docs/src/rules/sort-imports.md

```diff
@@ -193,29 +193,42 @@ import {b, a, c} from 'foo.js'
 ### `ignoreCase`
 
 When `true` the rule ignores the case-sensitivity of the imports local name.
+Uppercase letters of the alphabet must always precede lowercase letters.
 
-Examples of **incorrect** code for this rule with the `{ "ignoreCase": true }` option:
+Examples of **incorrect** code for this rule with the default `{ "ignoreCase": false }` option:
 
 ::: incorrect
 
 ```js
-/*eslint sort-imports: ["error", { "ignoreCase": true }]*/
-
-import B from 'foo.js';
+/*eslint sort-imports: ["error", { "ignoreCase": false }]*/
 import a from 'bar.js';
+import B from 'foo.js';
+import c from 'baz.js';
 ```
 
 :::
 
-Examples of **correct** code for this rule with the `{ "ignoreCase": true }` option:
+Examples of **correct** code for this rule with the default `{ "ignoreCase": false }` option:
 
 ::: correct
 
 ```js
-/*eslint sort-imports: ["error", { "ignoreCase": true }]*/
-
-import a from 'foo.js';
+/*eslint sort-imports: ["error", { "ignoreCase": false }]*/
 import B from 'bar.js';
+import a from 'foo.js';
+import c from 'baz.js';
+```
+
+:::
+
+Examples of **correct** code for this rule with `{ "ignoreCase": true }` option:
```

Comment:
its fine to keep the `incorrect` example for `{ "ignoreCase": true } ` that shows which case is not allowed by the option.

Reply:
Good suggestion. Thanks for the review!